packages/hoppscotch-sh-admin/src/helpers/userManagement.ts

```diff
@@ -0,0 +1,121 @@
+import { useToast } from '~/composables/toast';
+import { getI18n } from '~/modules/i18n';
+import { UserDeletionResult } from './backend/graphql';
+import { ADMIN_CANNOT_BE_DELETED, USER_IS_OWNER } from './errors';
+
+type IndividualActionInput = {
+  type: 'individual';
+  metadata: null;
+};
+type BulkActionInput = {
+  type: 'bulk';
+  metadata: {
+    areMultipleUsersSelected: boolean;
+    deletedIDs: string[];
+  };
+};
```

Comment:
Denotes the `action` arg that the helper function takes in.

- The `type` field denotes whether the action is performed from the `/users` page or via the `/user/_id` view.
- The `metadata` field includes additional information required based on the type. For instance, the `bulk` action type needs to convey if the action was performed on a single row or spanned multiple rows.